examples/tests/client-tests.js

```diff
@@ -0,0 +1,173 @@
+'use strict';
+
+fetch = () => {
+  return {
+    ok: true,
+    json() { return []; },
+  };
+};
+
+const baseMainChildrenNumber = document.querySelector('main').childElementCount;
+
+QUnit.module('Pictures');
+
+QUnit.test(
+  'putPicturesInPage',
+  (assert) => {
+    assert.ok(
+      typeof window.forTesting.putPicturesInPage === 'function',
+      '`putPicturesInPage` takes an array of picture objects and puts them as sections in <main>.',
+    );
+
+    const elM = document.querySelector('main');
+    assert.strictEqual(
+      elM.childElementCount,
+      (0+baseMainChildrenNumber),
+      'Before using the function, the element `main` must be empty of pictures.',
+    );
+    assert.ok(
+      document.querySelector('main .picture') == null,
+    );
+
+    // with an empty array
+    window.forTesting.putPicturesInPage([]);
+    const el = document.querySelectorAll('main section.picture');
+
+    assert.strictEqual(
+      el.length,
+      (0),
+      'When there is no picture to be added, there must be only the base code of the page.',
+    );
+
+    // with an array of one picture
+    let data = [{
+      id: 1,
+      file: 'http://placebear.com/400/200',
+      title: 'A bear',
+    }];
+    window.forTesting.putPicturesInPage(data);
+
+    assert.strictEqual(
+      elM.childElementCount,
+      (1+baseMainChildrenNumber),
+      'After passing one image entity to the function, there must be only one `section`.',
+    );
+
+    let listOfSections = elM.querySelectorAll('section.picture');
+    let section = listOfSections[0];
+    let paragraphs = section.querySelectorAll('p');
+    let as = section.querySelectorAll('a');
+    let images = as[0].querySelectorAll('img');
+    let deletes = section.querySelectorAll('div');
+
+    assert.ok(
+      section.classList.contains('picture'),
+      'A `section` must have the class `picture`.',
+    );
+
+    assert.strictEqual(
+      paragraphs.length,
+      1,
+      'There must be one `<p>` element in a `section`.',
+    );
+    assert.strictEqual(
+      paragraphs[0].className,
+      'title',
+      'The first paragraph must have the class `title`',
+    );
+
+    assert.strictEqual(
+      as.length,
+      1,
+      'There must be one `<a>` element in a `section`.',
+    );
+    assert.ok(
+      as[0].classList.contains('img'),
+      'The `<a>` element must have the class `img`.',
+    );
+    assert.strictEqual(
+      as[0].href,
+      data[0].file,
+      'The `<a>` element must have its `href` attribute set to the good value.',
+    );
+
+    assert.strictEqual(
```

Comment:
we have a blank line before this block of tests, maybe add another line with a comment like 
```
// test the image inside the a
```